analyses/independent-samples/run-independent-samples.sh

```diff
@@ -22,3 +28,14 @@ Rscript 02-generate-independent-rnaseq.R \
   --output_directory results \
   --independent_dna_sample_df ../../data/independent-specimens.wgswxs.primary-plus.tsv  
 
+else 
+Rscript 01-generate-independent-specimens.R \
+  -f ../../data/pbta-histologies-base.tsv \
```

Comment:
It looks like the only thing that is different is whether `../../data/pbta-histologies-base.tsv`or `../../data/pbta-histologies.tsv` is used, so a shorter way to do this is to use a smaller if/then specify a `HISTOLOGIES_FILE` variable and supply that to the `-f` and `--histology_file` options. Then you can get rid of this larger if/then and not have to repeat these Rscript calls.

Reply:
ah that's a great idea, I'll update that, thanks!